analyses/molecular-subtyping-HGG/00-HGG-select-pathology-dx.R

```diff
@@ -27,9 +33,21 @@ exact_path_dx<- c(
 # as `anaplastic gliomatosis cerebri (who grade 4)`
 gliomatosis_path_free_text_exact <- "anaplastic gliomatosis cerebri (who grade 4)"
 
+#Identify which samples are IHGs and bring them into the HGG module for subtyping. 
+#This can be done by either searching 
+#pathology_free_text_diagnosis for the term/terms: "infant type hemispheric glioma" or 
+#cns_methylation_subclass == "IHG".
+
+IHG_Participant_ID <- histo %>%
+  filter(cns_methylation_subclass == "IHG" | grepl("infant type hemispheric glioma", pathology_free_text_diagnosis)) %>%
+  pull(Kids_First_Participant_ID) %>%
+  unique() 
```

Comment:
We will want to do this on a tumor level - patients can have more than one tumor type (for ex, LGG can upgrade to HGG or there can be a secondary tumor).

I might suggest using the JSON file to just add these terms - a new one for the pathology free text dx and another one for the methylation subtype call